generate_tensorrt.bash

```diff
@@ -1,25 +1,41 @@
 #!/bin/bash
 config="$1"
+run_on_jetson="$2"
```

Comment:
Make sure this is also changed in the docs and readme

Reply:
Sure, Thanks